sdk/src/main/scala/com/horizen/block/WithdrawalEpochCertificate.scala

```diff
@@ -13,7 +14,16 @@ import com.horizen.librustsidechains.{Utils => ScCryptoUtils}
 import scala.util.Try
 
 case class FieldElementCertificateField(rawData: Array[Byte]) {
-  lazy val fieldElementBytes: Array[Byte] = {
+  def fieldElementBytes(version: SidechainCreationVersion): Array[Byte] = {
+    version match {
+      case SidechainCreationVersion0 =>
+        // prepend raw data to the FieldElement size
+        Bytes.concat(new Array[Byte](FieldElementUtils.fieldElementLength() - rawData.length), rawData)
+      case SidechainCreationVersion1 =>
+        // append raw data to the FieldElement size
+        Bytes.concat(rawData, new Array[Byte](FieldElementUtils.fieldElementLength() - rawData.length))
+      case other => throw new IllegalArgumentException(s"Version $other is not supported.")
+    }
     Bytes.concat(new Array[Byte](FieldElementUtils.fieldElementLength() - rawData.length), rawData)
```

Comment:
This line should be removed

Reply:
fine, will leave it here as is, because was fixed in https://github.com/HorizenOfficial/Sidechains-SDK/pull/333